lib/SPIRV/SPIRVWriter.cpp

```diff
@@ -4284,7 +4285,8 @@ void LLVMToSPIRVBase::transGlobalAnnotation(GlobalVariable *V) {
         cast<GlobalVariable>(CS->getOperand(1)->stripPointerCasts());
 
     StringRef AnnotationString;
-    getConstantStringInfo(GV, AnnotationString);
+    bool HasAnno = getConstantStringInfo(GV, AnnotationString);
+    assert(HasAnno && "Annotation string missing");
```

Comment:
```suggestion
    assert(HasAnno && "Annotation string missing");
    (void)HasAnno;
```

Or non-assert `-Werror` build will fail. Alternatively, `std::ignore = HasAnno;` can be used

Reply:
Fixed it in a different way. Thanks